game/script-idles.rpy

```diff
@@ -594,3 +602,39 @@ label idle_laptop:
     $ jnPause(1)
 
     $ jn_idles._concludeIdle()
+
+label idle_math_attempts:
+    show black zorder JN_BLACK_ZORDER with Dissolve(0.5)
+    show prop math_attempt zorder JN_PROP_ZORDER
+    show natsuki working_on_papers
+    hide black with Dissolve(0.5)
+    $ jnClickToContinue(silent=False)
+
+    if random.choice([True, False]):
+        n 1tnmboeqm "...?{w=1.25}{nw}"
+        n 1uwdajesu "Oh!{w=0.75}{nw}"
+        extend 1fchbgsbl " H-{w=0.2}Hey [player]."
```

Comment:
```suggestion
        extend 1fchbgsbl " H-{w=0.2}hey [player]."
```

Forgot last time, but we don't usually capitalise the follow-up character to the stutter.